advanced/accessibility.md

```diff
@@ -11,27 +11,36 @@ keywords: accessibility wai aria jaws nvda
 
 ## Keyboard shortcuts
 
-The `Alt+key` shortcut assumes that you are currently in the content area of the editor.
+The **Alt+key** shortcut assumes that the cursor is in the content area of the editor.
 
 | Alt Key   | Result      |
 |-----------|-------------|
-| Alt + F9  | Focus/jump to menu bar |
-| Alt + F10 | Focus/jump to toolbar |
-| Alt + F11 | Focus/jump to element path |
+| Alt+F9  | Focus/jump to menu bar |
```

Comment:
Is it made clear somewhere that Alt means option for macOS users like myself?

Reply:
:facepalm: sort-of, but it should be duped here.

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1834/commits/9179e84030d4f64333b2f0862f175f55b312c9a6